### ISSUE TITLE
Notify TOs about new Registrations

### ISSUE THREAD

**Issue by 5N00P1** [madarasz/always-be-running#51]
It's nice as a TO to know, when some one new applies to the tournament. So getting a notification would be nice.

On top it would be nice to switch this off, when you are creating the tournament for some one else. As it happened for Euros and I don't want to get all notifications for the German Nationals!

**Comment by madarasz** [comment]
This is currently low prio, it's easy for a TO to check registered users before the tournament starts.

Yep, a notification system (email/browser/slack) would be nice, it's on my list.

**Comment by 5N00P1** [comment]
I would found it useful as a TO to be honest as I have an idea of when a new registration is done so I have a feeling for the number of attendees without checking the page so often.

**Comment by 5N00P1** [comment]
This time I got 3 cancellations on the day before and one new registration. For a tournament of 14 people that was ok, for more people this will get confusing when you try to use ABR as registration platform.

**Comment by madarasz** [comment]
Yep, I see. Alas, I still have to figure out some email certificate/security/authenticatoin stuff, because although I can send emails but Gmail just bounces my stuff back :(
